torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -68,6 +70,8 @@ class _X86InductorQuantizationAnnotation(QuantizationAnnotation):
     torch.ops.aten.max_pool2d.default,
     torch.ops.aten.cat.default,
     torch.ops.aten.avg_pool2d.default,
+    torch.ops.aten.adaptive_avg_pool2d.default,
+    torch.ops.aten.flatten.using_ints,
```

Comment:
Is `int8_in_int8_out_ops_pt2e` also `quantizable_ops_pt2e`? If so, we can perhaps define `quantizable_ops_pt2e` with `int8_in_int8_out_ops_pt2e` plus additions?

Reply:
Thank you, yes, I agree. `int8_in_int8_out_ops_pt2e` should ultimately be a subset of `quantizable_ops_pt2e` (currently, these two sets are equal). Therefore, we can define `quantizable_ops_pt2e` by including `int8_in_int8_out_ops_pt2e` and additional operators.